src/test/scala/zio/inspections/SimplifyZipRightInspectionTest.scala

```diff
@@ -117,3 +117,29 @@ class SimplifyFlatmapWithZipRightOperatorTest extends ZipRightInspectionTest("*>
     z(s"""ZIO.succeed("Benito Quinquela Martín").${START}flatMap(x => x)$END""").assertNotHighlighted()
 
 }
+
+class SimplifyZipRightToSucceedInspectionTest extends ZSimplifyInspectionTest[SimplifyZipRightToSucceedInspection] {
+  override protected val hint = s"Replace with .as"
+
+  def test_zipRight_to_succeed(): Unit = {
+    z(s"""${START}f("Fernando Fader").zipRight(ZIO.succeed("Carlos Alonso"))$END""").assertHighlighted()
+    val text   = z("""f("Fernando Fader").zipRight(ZIO.succeed("Carlos Alonso"))""")
+    val result = z("""f("Fernando Fader").as("Carlos Alonso")""")
+    testQuickFixes(text, result, hint)
+  }
+
+  def test_zipRight_infix_invocation_to_succeed(): Unit = {
+    z(s"""${START}f("Fernando Fader") zipRight ZIO.succeed("Carlos Alonso")$END""").assertHighlighted()
+    val text   = z("""f("Fernando Fader") zipRight ZIO.succeed("Carlos Alonso")""")
+    val result = z("""f("Fernando Fader").as("Carlos Alonso")""")
+    testQuickFixes(text, result, hint)
+  }
+
+  def test_zipRight_operator_to_succeed(): Unit = {
+    z(s"""${START}f("Fernando Fader") *> ZIO.succeed("Carlos Alonso")$END""").assertHighlighted()
+    val text   = z("""f("Fernando Fader") *> ZIO.succeed("Carlos Alonso")""")
+    val result = z("""f("Fernando Fader").as("Carlos Alonso")""")
+    testQuickFixes(text, result, hint)
+  }
+
```

Comment:
nit: we really need this newline?

Reply:
My thoughts — nope, but that's the style other tests are written.